src/Stratis.StratisD/Properties/launchSettings.json

```diff
@@ -1,7 +1,8 @@
 {
   "profiles": {
     "Stratis.StratisD": {
-      "commandName": "Project"
+      "commandName": "Project",
```

Comment:
the `Stratis.StratisD Test` launch setting already has `-testnet`.

Reply:
sorry my mistake, i did not want to commit this file, it happens so often :/ would be great to have it in git ignore 

will remove from PR